packages/blockchain-extension/cucumber/helpers/smartContractHelper.ts

```diff
@@ -123,7 +123,7 @@ export class SmartContractHelper {
         return contractDirectory;
     }
 
-    public async packageSmartContract(name: string, version: string, language: string, directory: string): Promise<PackageRegistryEntry> {
+    public async packageSmartContract(name: string, version: string, language: string, directory: string, fabricVersion: any): Promise<PackageRegistryEntry> {
```

Comment:
Can we give `fabricVersion: any` a proper type and also make it possible so we don't have to do `const sanitisedFabricVersion: number = (fabricVersion === 1 || fabricVersion === 'v1' || fabricVersion === '1') ? 1 : 2;`.
I think it would be good to standardise this function so we can only call it one way.

Reply:
We already have `UserInputUtil.V1_4_2` and `UserInputUtil.V2_0`, could we use them here?

Reply:
I can remove the `fabricVersion === 1` as that shouldn't be hit but I'll leave `'v1's and `'1'` as it acts as a failsafe for the cucumber tests so that they work whether we put "Fabric 1" or "Fabric v1".